atom_core/src/atom_core/config_io.py

```diff
@@ -16,6 +16,25 @@
 from atom_core.utilities import atomError
 from atom_core.system import resolvePath, expandToLaunchEnv
 
+def mutually_inclusive_conditions(A,B):
+
+    if A is not None and B is not None:
+        return True
+    elif A is None and B is None:
+        return False
+    else:
+        atomError(f'{Fore.RED}-ntfl{Style.RESET_ALL} and {Fore.RED}-ntfv{Style.RESET_ALL} flags are mutually inclusive') 
+        exit()
```

Comment:
I do not think this is mutually inclusivity. Mutual inclusivity is that two conditions **can** occur at the same time. [(source)](https://math.stackexchange.com/questions/2356299/whats-the-difference-between-independent-event-and-mutually-inclusive-event)
This is more related [XNOR gate](https://en.wikipedia.org/wiki/XNOR_gate), but is not the same as well. I understand what you want to do, but please rename it.

Reply:
I struggled for a while with this one.

The behavior I was looking for is : 

- If both A and B are defined, execute a function;
- If neither A nor B are defined, don't execute a function and continue;
- If only one is defined, trigger a error.

Either way, from the example your source gave, I still think I'm right. Looking at the example : 

> Example of mutually inclusive events: suppose you roll a six-sided die wanting to get the number 6 on the top. Then the number 1 must be on the bottom. Thus, one cannot happen without the other occurring. 

Number 1 on bottom and number 6 on top are events that **can** happen, but not independently. If one happens, the other also has to happen. You can't get number 1 on the bottom without 6 on top. You can however get neither.

I think the definition you gave should be instead : 

- Mutual inclusivity is two conditions that if they occur, both MUST occur at the same time.

Which I think is precisely what I have. Both flags can occur at the same time, they can both not occur, but one can not occur if the other does.

I don't think its a XNOR gate as if both are false, xnor returns true. Here if both are false it should return false. Also I'm not sure how to properly express this but a XNOR gate can only output True or False, but all cases are possible. 
If 2 events are mutually inclusive (in the way I see it), they can both not happen, both happen but just one happening isn't possible. I need a function that can output these 3 cases, like the one I have.

Also considering mutually incu